pkg/engine/supervisor.go

```diff
@@ -549,6 +549,11 @@ func (e *Engine) doRun(ctx context.Context, id string, input *RunInput, ow *rpc.
 	// 2. Get the env config for the runner.
 	cfg = cfg.Append(e.envcfg.Runners[trunner])
 
+	var flag = cfg[0][config.RunnerDisabledFlag]
```

Comment:
This access pattern is going to be confusing, since we're dealing with a special, global flag, what about making it clear this is not a configuration we're going to compile, but actually a global runner config?

```suggestion
	var flag = e.envcfg.Runners[trunner][config.RunnerDisabledFlag]
```